### ISSUE TITLE
HTTP Logging Middleware

### ISSUE THREAD

**Issue by dodyg** [dodyg/practical-aspnetcore#286]
https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#http-logging-middleware

**Comment by lohithgn** [comment]
should this be in ```projects > net6 > middlewares > http-logging``` ? what was your idea ?

**Comment by dodyg** [comment]
Yup. We need a couple of samples for this and see if there are advanced scenario that's not covered in documentation yet.
